packages/node-preload/src/index.ts

```diff
@@ -109,9 +112,12 @@ async function loadPreload() {
   const webPreloadPath = path.join(__dirname, "web-preload.js");
   const webPreload = fs.readFileSync(webPreloadPath, "utf8");
   await webFrame.executeJavaScript(webPreload);
+
+  const func = await webFrame.executeJavaScript("async () => { await window._moonlightBrowserLoad(); }");
```

Comment:
We should probably rename this to "moonlightWebLoad" now that it's used by both entrypoints.